### ISSUE TITLE
false positive in no-unused-variable rule

### ISSUE THREAD

**Issue by HamletDRC** [palantir/tslint#814]
This code triggers a violation within the no-unused-variable rule. It claims StreamConstants is unused. 

```ts
import StreamConstants = require('social-center/constants/StreamConstants');
import StreamState = StreamConstants.State;
```

The 2nd line clearly uses StreamConstants.

**Comment by adidahiya** [comment]
dupe of #325
